### DISCUSSION TITLE
inputtext: add `clear` icon for

### DISCUSSION THREAD

**Discussion by nimo23** [primefaces/community#3126] [PrimeReact]
The dropdown has a `clear` icon (https://primereact.org/dropdown/#clearicon):

> Clear Icon
> 
> When showClear is enabled, a clear icon is added to reset the Dropdown.

Would be nice to support this clear icon also for https://primereact.org/inputtext/, e.g.:

`<inputText showClear`

**Comment by melloware** [comment]
Similar request was made for Calendar: https://github.com/primefaces/primereact/issues/6745
